tests/test_tarfile.py

```diff
@@ -0,0 +1,47 @@
+import tarfile
+
+with tarfile.open("test.tar.xz", "w:xz") as tar:
+    pass
+
+# Test with valid preset values
+for preset in range(10):
+    with tarfile.open("test.tar.xz", "w:xz", preset=preset) as tar:
+        pass
```

Comment:
We just need to verify that this works with `int` values:

```suggestion
tarfile.open("test.tar.xz", "w:xz", preset=0)
tarfile.open("test.tar.xz", "w:xz", preset=5)
tarfile.open("test.tar.xz", "w:xz", preset=9)
```

Reply:
adjusted.